test/skimNtuple_HHbtag.cpp

```diff
@@ -5039,6 +5071,13 @@ int main (int argc, char** argv)
 			  ++theSmallTree.m_njets ;
 			} // loop over jets
 
+		  // only apply HEM1516 veto to specific runs in data + corresponding fraction of MC
+		  TRandom3 *rand = new TRandom3();
+		  if((!isMC and theBigTree.RunNumber < 319077) or
+		     (isMC and rand->Rndm(1234) < 0.6347)) HEM1516veto = 0; // randomly selecting 63.47% of MC events (fraction of total lumi affected by HEM issue
```

Comment:
- I think, because you recreate the TRandom3 here for each event, the "random" number will always be the same. 
- If 63.47% of the 2018 lumi is affected by the HEM issue, shouldn't the `<` be a `>`?
- Is the idea to only check this variable for 2018? Because this way also this fraction of 16 and 17 MC events will have HEM1516veto = 1, correct?

Reply:
3 very good points, thanks! fixes and a test on MC are coming

Reply:
should be fixed in https://github.com/LLRCMS/KLUBAnalysis/pull/391/commits/fa2053354e2f132bc27cd0f145b2728713e93711. Running on some 2018 signal MC I do get coherent results (effect on yield of ~ 4% for the full year -> can extrapolate to ~7% for the affected period, compatible with observation in data)